apps/judicial-system/backend/src/app/formatters/pdf.ts

```diff
@@ -453,6 +453,17 @@ export async function generateRulingPdf(existingCase: Case): Promise<string> {
     )
     .text(' ')
     .font('Helvetica-Bold')
+    .text(
+      `${
+        existingCase.registrar?.name || 'Dómritari hefur ekki verið skráður'
+      } ${existingCase.registrar?.title || ''}`,
```

Comment:
According to the Asana task, the registrar should not show up here. This is the signature line of the judge.

The registrar should be autofilled in the courtAttendees field in the client. That field is already in the pdf, so no need to add anything here. See:
![image](https://user-images.githubusercontent.com/795382/107761383-2f503280-6d23-11eb-96d3-c06c269c2a6a.png)
![image](https://user-images.githubusercontent.com/795382/107761323-16478180-6d23-11eb-90d0-872f4a4fcebe.png)

Reply:
Good eye! Thank you 🙏